ios/RNWootric.m

```diff
@@ -1,17 +1,32 @@
 #import <WootricSDK/WootricSDK.h>
+#import <WootricSDK/WTRSurveyDelegate.h>
 #import "RNWootric.h"
 
 @implementation RNWootric
 
+bool hasListeners;
```

Comment:
I'm a bit rusty, but strictly speaking, you probably want this to be a property on the `RNWootric` instance (even though it's likely a singleton).

So rather than `bool hasListeners;` here, in the `RNWootric.h` file put `@property (nonatomic, assign) BOOL hasListeners;`.

Then to access it in the implementation, it's `self.hasListeners` rather than just `hasListeners`.

Reply:
(FYI this came from https://reactnative.dev/docs/native-modules-ios#sending-events-to-javascript but was adapted a bit)

Reply:
Interesting! It's not a blocker in any case. It just means the variable is not scoped to the object, but it's not going to have an impact ultimately.